### ISSUE TITLE
Member's site

### ISSUE THREAD

**Issue by nhyty** [newtheatre/website#10]
Decide on what it should be and update all information

**Comment by wjdp** [comment]
Can this be a thing? As has been seen it's a very useful resource for the future when stuff is added to it and people know it's there :weary:

**Comment by pickleshb** [comment]
Make it a nice looking static site (maybe hugo) can edit pages with [prose.io](prose.io)

**Comment by wjdp** [comment]
I would look to maintain it as a wiki to keep the barrier to entry low for editors. If there is a better solution than DokuWiki I'm open to suggestions. I was tempted to fiddle with the template when setting up but it's actually very well (functionally) designed.

Having a quick browse now I did feel a little lost at times, perhaps would be an idea to implement an expanding tree sitemap in the sidebar?

**Comment by wjdp** [comment]
From a non-technical standpoint what needs doing is evaluating what could be on there. The more useful -- and relevant -- information on there will increase usage.

**Comment by pickleshb** [comment]
literally everything's better than DokuWiki, I feel like if you are going to be inclined to edit the wiki then you can take the time to maybe learn a little bit of markdown

**Comment by wjdp** [comment]
I'd prefer it to use markdown, would be very nice. My reasons for liking/choosing were:

- NT user accounts, doku is used for single sign in (albeit only with room bookings)
- Ease of editing, the editor is on the same site, looks the same, no external service
- Less work to set up, little/no dev work
- Access controls, committee section (superseded by Google Drive)

Re. prose, getting everyone on GitHub feels like a larger task than signups on a self-hosted wiki site.

**Comment by pickleshb** [comment]
We could move the room booking site somewhere and set it up to use Github OAuth to authenticate members, then the members' site and the room booking site is on the same account for people

**Comment by wjdp** [comment]
Feels like we're going down a rabbit hole of things that aren't broken but would be nice to make anew. We should have this conversation in a meeting and focus here on what we should be doing now. @nhyty can you chime in?

**Comment by pickleshb** [comment]
I agree, focus on the main site first then make everything else nice and new

**Comment by nhyty** [comment]
I think the plan for this is to be compiled into several packs, much info already has been, and then this site to be retired :(
